content/en/agent/guide/agent-fips-proxy.md

```diff
@@ -0,0 +1,348 @@
+
+The `https` option is set to `false` because the Agent uses HTTP to communicate with the proxy. The Datadog Agent FIPS Proxy is expected to run on the same host as the Datadog Agent and relies on the host's security for protection of that communication.
+
+**Host security and hardening are customer responsibilities.**
+
+<div class="alert alert-warning">The <code>fips.enabled</code> setting defaults to <code>false</code> in the Agent. It must be set to <code>true</code> to ensure all communications are forwarded through the Datadog Agent FIPS Proxy.<br><br><strong>If <code>fips.enabled</code> is not set to <code>true</code>, the Agent is not FIPS Compliant</strong>.</div>
+
+### Verify your installation
+
+Verify that metrics, traces, and logs are correctly reported in the app.
+
+For metrics, run the connectivity diagnostic command and verify that all checks pass:
+
+```shell
+sudo -u dd-agent datadog-agent diagnose datadog-connectivity
+```
+
+If you're not seeing metrics, traces, or logs reported in the app, see the [Troubleshooting](#troubleshooting-a-bare-metal-or-vm-installation) section.
+
+### View logs
+
+```shell
+sudo journalctl -u datadog-fips-proxy
+```
+
+#### journald logs configuration
+
+If you're using [Log Management][3] and want to send the Datadog Agent FIPS Proxy logs to Datadog, you must set up the Datadog Agent to read logs from journald.
+
+1. In the Agent's [configuration file][2], set `logs_enabled` to `true` to activate the Logs Agent. In the [configuration directory][4], create a file at `fips_proxy.d/conf.yaml` with the following content:
+
+   ```yaml
+   logs:
+     - type: journald
+       source: datadog-fips-proxy
+       include_units:
+         - datadog-fips-proxy.service
+   ```
+
+1. Make sure that the `dd-agent` user is in the `systemd-journal` group. For more information, see the [journald integration][5] documentation.
+1. [Restart the Agent][6].
+
+[1]: https://app.datadoghq.com/account/settings#agent/
+[2]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
+[3]: /logs/
+[4]: /agent/guide/agent-configuration-files/#agent-configuration-directory
+[5]: /integrations/journald/#configuration
+[6]: /agent/guide/agent-commands/#start-stop-and-restart-the-agent
+{{% /tab %}}
+
+{{% tab "Helm on Amazon EKS" %}}
+Set the following values in your `values.yaml` file:
+
+```yaml
+fips:
+  enabled: true
+  use_https: false
+```
+
+The `fips` setting is available in Agent versions >= 7.41. When the setting is enabled, the Datadog Agent redirects all of its communications to the Datadog Agent FIPS Proxy. This setting ignores custom URL options, such as `dd_url`.
+
+The `use_https` option is set to `false` because the Agent uses HTTP to communicate with the proxy. The Datadog Agent FIPS Proxy is expected to run on the same host as the Datadog Agent and relies on the host's security for protection of that communication. 
```

Comment:
```suggestion
The `use_https` option is set to `false` because the Agent uses HTTP to communicate with the proxy. The Datadog Agent FIPS Proxy runs on the same host as the Datadog Agent and relies on the host's security for protection of that communication. 
```